frontend/cypress/integration/common/istio_config.ts

```diff
@@ -1,6 +1,7 @@
 import { After, Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
 import { colExists, getColWithRowText } from './table';
 import { ensureKialiFinishedLoading } from './transition';
+import { getGVKTypeString } from '../../../src/utils/IstioConfigUtils';
```

Comment:
Could you use absolute 'utils' folder path in cypress test imports? Relative paths do not work well with OSSMC.

```suggestion
import { getGVKTypeString } from 'utils/IstioConfigUtils';
```

Reply:
sure, done :)